src/components/AvatarWithIndicator.js

```diff
@@ -10,24 +10,33 @@ const propTypes = {
 
     // url for the avatar
     source: PropTypes.string.isRequired,
+
+    // avatar size
+    size: PropTypes.string,
+
 };
 
 const defaultProps = {
     isActive: false,
+    size: 'default',
 };
 
 const AvatarWithIndicator = ({
     isActive,
     source,
+    size,
 }) => {
     const indicatorStyles = [
-        styles.statusIndicator,
+        size === 'large' ? styles.settingsStatusIndicator : styles.statusIndicator,
```

Comment:
This should also be referencing a more generic style name and not the settings page. Nothing should be referencing settings in this file.